app/controllers/parent_interface/consent_forms/edit_controller.rb

```diff
@@ -39,14 +39,25 @@ def update
         )
       end
 
-      if current_step == :parent &&
-           @consent_form.parental_responsibility == "no"
-        return(
-          redirect_to session_parent_interface_consent_form_cannot_consent_responsibility_path(
-                        @session,
-                        @consent_form
-                      )
+      if current_step == :parent
+        if @consent_form.parental_responsibility == "no"
+          return(
+            redirect_to session_parent_interface_consent_form_cannot_consent_responsibility_path(
+                          @session,
+                          @consent_form
+                        )
+          )
+        end
+
+        # rename keys, taking parent_ out of the key
+        parent_params =
+          update_params
+            .except(:form_step)
+            .transform_keys { |key| key.to_s.gsub("parent_", "") }
```

Comment:
Sorry to nitpick, but:

```suggestion
            .transform_keys { |key| key.to_s.sub("parent_", "") }
```

Reply:
ah, sorry, this bit gets refactored away to something much simpler in a later commit, so safe to ignore.